cmd/create/cluster/cmd.go

```diff
@@ -507,6 +506,8 @@ func run(cmd *cobra.Command, _ []string) {
 				os.Exit(1)
 			}
 		}
+	} else {
+		validations.Validations(cmd, []string{})
```

Comment:
do i need to push it at the end of all the inputs

Reply:
no, you can probably just remove that Validations function and call it inside `run` like:
```
if args.roleARN != "" {
  validations.Validations()
}
```
maybe it needs a different call signature (instead of passing `cmd`, you can just pass the AWS Client to find the region tag with)

Reply:
and do that at the begining of all the arguments, before the user enters anything, since the `roleARN` needs to be explicit